src/account/utils.ts

```diff
@@ -1,5 +1,31 @@
+import { parsePhoneNumber } from '@celo/phone-utils'
 import { ADDRESS_LENGTH } from 'src/exchange/reducer'
 
 export const isAddressFormat = (content: string): boolean => {
   return content.startsWith('0x') && content.length === ADDRESS_LENGTH
 }
+
+export function getPhoneNumberState(
```

Comment:
optional nit: consider renaming State to something else, I think it made sense when it was within the component file, but not super obvious in utils.